cloudevents/extensions/loglevel.md

```diff
@@ -0,0 +1,167 @@
+# Log Level Extension
+
+## Abstract
+This extension defines attributes that MAY be included within a CloudEvent
+to describe the "level" or "importance" of an event in relation to other events
+produced by a unique event source.
+
+Often systems produce events in form of logs, and these types of events usually share
+a common concept of "log-level". This extension aims to provide a standard way for 
+describing this property in a language agnostic form. 
+
+Sharing a common way to describe importance of events allows for better monitoring 
+systems, tooling and general log consumption.
+
+In addition to defining the extension attributes, this document defines 
+the concept of [Log Level Bindings](#log-level-bindings) - a standard way to map 
+well-known log level names from different languages and frameworks 
+onto the `loglevelname` and `loglevelnum` values defined in this spec. 
+
+## Attributes
+
+All attributes of this extension are OPTIONAL. however, when using this 
+custom extension you MUST use at-least one of these OPTIONAL attributes. 
+
+### loglevelname 
+
+- Type: `String`
+- Description: Human readable representation of the log level
+- Constraints
+  - OPTIONAL
+  - if present, MUST be a non-empty string
+  - SHOULD be lowercase
+  - RECOMMENDED values include `emergency`, `alert`, `critical`, `error`, 
```

Comment:
"include" => "are" - those are the only values we *recommend*, but others may be used.